app/assets/locales/de/allergen.json

```diff
@@ -1,5 +1,5 @@
 {
-  "allergenTitle": "Allergene:",
```

Comment:
maybe also rename the key?

Reply:
maybe, but as i might have changes intersecting between this branch and the envinfo branch i won't change it now to not break anything when merging